### DISCUSSION TITLE
How to Send Encrypted Email via Postal

### DISCUSSION THREAD

**Discussion by absolutezack** [postalserver/postal#2215] [Help with using Postal]
Hi Guys,

I manage several postal servers. I didn't do any configuration related certificates on all my postal servers. But I found on one of the servers (let's say server A), there is a message like the following when sending an email to Gmail:
`from: test@domain1.com
to: bankjek48@gmail.com
date: Oct 25, 2022, 2:27 PM
subject: SUBJECT EMAIL
mailed-by: psrp.domain1.com
signed-by: domain1.com
**security: Standard encryption (TLS) [Learn more](https://support.google.com/mail?hl=en&p=tls)**`

While on the other server (let's say server B), there is a message like the following when sending an email to Gmail accompanied by a red icon:
`from: test@domain2.com
to: bankjek48@gmail.com
date: Nov 3, 2022, 12:28 PM
subject: SUBJECT EMAIL
mailed-by: psrp.domain2.com
signed-by: domain2.com
**security: **No encryption** Learn more**`

Then I found the following in the raw email message sent via server A:
`Received: from servera.domain1.com ([IP ADDRESS])
        by mx.google.com with ESMTPS id r64-20020a632b43000000b00439f012ca81si1772668pgr.605.2022.10.25.00.27.12
        for <bankjek48@gmail.com>
        (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);`

Meanwhile, there is no such section in the raw email message sent via server B.

How do I get messages sent via server B to be encrypted like messages sent via server A?
Thankyou in advance

**Comment by willpower232** [comment]
Have you enabled TLS in the SMTP server settings?

https://github.com/postalserver/postal/blob/83fef0e8a01285e6c9cb839d72d2cfca2371c839/config/postal.defaults.yml#L68-L70

**Reply by absolutezack** [reply]
I've already done that on the postal host (/opt/postal/config/postal.yml). But when I check via openssl, I got this error:
write:errno=104
no peer certificate available
No client certificate CA names sent
SSL handshake has read 134 bytes and written 316 bytes
Verification: OK

**Reply by willpower232** [reply]
Have you restarted Postal since changing the config file? Have you definitely given Postal the certificate files? You would need to put them into the config directory or update the docker-compose.yml file to include them.
